src/editor/ui_menus/toolsize_menu.cc

```diff
@@ -33,38 +33,22 @@ inline EditorInteractive& EditorToolsizeMenu::eia() const {
 EditorToolsizeMenu::EditorToolsizeMenu(EditorInteractive& parent,
                                        UI::UniqueWindow::Registry& registry)
    : UI::UniqueWindow(
-        &parent, UI::WindowStyle::kWui, "toolsize_menu", &registry, 250, 50, _("Tool Size")),
-     textarea_(this,
-               UI::PanelStyle::kWui,
-               UI::FontStyle::kWuiLabel,
-               5,
-               10,
-               240,
-               10,
-               std::string(),
-               UI::Align::kCenter),
-     increase_(this,
-               "incr",
-               get_inner_w() / 2 - 10,
-               25,
-               20,
-               20,
-               UI::ButtonStyle::kWuiSecondary,
-               g_image_cache->get("images/ui_basic/scrollbar_up.png")),
-     decrease_(this,
-               "decr",
-               get_inner_w() / 2 + 10,
-               25,
-               20,
-               20,
-               UI::ButtonStyle::kWuiSecondary,
-               g_image_cache->get("images/ui_basic/scrollbar_down.png")),
+        &parent, UI::WindowStyle::kWui, "toolsize_menu", &registry, 250, 30, _("Tool Size")),
+     toolsize_spinbox_(this,
+                       5,
+                       5,
+                       get_inner_w() - 2 * 5,
```

Comment:
consider using UI::Box for proper positioning while avoiding magic numbers and manual calculation

Reply:
As far as I can tell from looking at tool_change_height_options.cc, I have to set margins if I use UI::Box too. Anyhow, at least I was able to make the size adapt to the length of the text in case some translation is longer.

I'm sure using a const member for margin is all wrong though (it seems to me that anything I'm able to parse and understand is considered bad practice in C++... ;) so please let me know what I should use instead.